README.md

```diff
@@ -1,4 +1,4 @@
-# r-cubed: TODO TITLE <img src="_extensions/r3-theme/favicon.ico" align="right" height="100"/>
+# Collaboration with Git and GitHub: A gentle introduction to a team-based collaborative workflow using Git and GitHub
```

Comment:
```suggestion
# Collaboration with Git and GitHub: A gentle introduction to a collaborative workflow using Git and GitHub
```
Is it maybe a bit redundant to include both "team-based" and "collaborative" in this quite long header?

Reply:
Technically, "collaborate" doesn't require being in a team. You can collaborate together as a group for something, but not in the context of being a team working together. In this sense, it would make more sense to drop collaborate and keep language about being a team. But "to team" isn't a verb :stuck_out_tongue:

Reply:
This is also outside the scope of this PR and relates to the #20 issue. So let's ignore the details of the title and I'll make an issue about it

Reply:
team-based workflow, then?

Reply:
But you're right, it's outside the scope of this PR..